plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableRefundDataStore.php

```diff
@@ -143,6 +143,7 @@ public function create( &$refund ) {
 	 */
 	public function update( &$refund ) {
 		$this->persist_updates( $refund );
+		$this->clear_caches( $refund );
```

Comment:
Not sure if this call is necessary: `persist_updates` will clear the cache due to `$backfill = true` fallback. Can you please verify?

Reply:
Nice! I'm reviewing and manually testing this PR.